### PR TITLE
build: switch to patched Windows image with Node 12

### PR DESCRIPTION
**Pull request by bcoe**
While applying patches to Windows image, I also fixed up some path issues.

### CODE REVIEW

**Comment by bcoe** [comment]
I'd like to create the manual PR for this before landing.

File: synthtool/gcp/templates/node_library/.kokoro/test.bat [hunk 1]
```diff
@@ -17,14 +17,12 @@
...
-@rem by nvm (v10.15.3), which has no npm bin. This hack uses the functional
-@rem Node v8.9.1 to install npm@latest, it then uses this version of npm to
-@rem install npm for v10.15.3.
-call nvm use v8.9.1 || goto :error
-call node C:\Users\kbuilder\AppData\Roaming\nvm-ps\versions\v8.9.1\node_modules\npm-bootstrap\bin\npm-cli.js i npm -g || goto :error
-call nvm use v10.15.3 || goto :error
-call node C:\Users\kbuilder\AppData\Roaming\nvm-ps\versions\v8.9.1\node_modules\npm\bin\npm-cli.js i npm -g || goto :error
+@rem npm path is not currently set in our image, we should fix this next time
+@rem we upgrade Node.js in the image:
+SET PATH=%PATH%;/cygdrive/c/Program Files/nodejs/npm
```
Comment (chingor13) [inline synthtool/gcp/templates/node_library/.kokoro/test.bat]: I feel like this should be done in our image. This is a lot of knowledge for the build to know about its windows environment.

Comment (bcoe) [reply synthtool/gcp/templates/node_library/.kokoro/test.bat]: Could I convince you that I do this next time I update the image, it's pretty annoying to delete and snapshot the image a second time this week.

Comment (bcoe) [reply synthtool/gcp/templates/node_library/.kokoro/test.bat]: (also this is a vast improvement to the backflips we were doing to get node v10.15.3 running in `test.bat` prior to this).

**Pull request merged**